docs/nx-cloud/reference/launch-templates.md

```diff
@@ -67,26 +321,21 @@ The following images are available:
 - `ubuntu22.04-node20.9-withDind-v2`
 - `ubuntu22.04-node20.9-v3`
 - `ubuntu22.04-node20.9-withDind-v3`
+- `ubuntu22.04-node20.9-withDind-v4`
+- `ubuntu22.04-node20.9-withDind-v5`
+- `ubuntu22.04-node20.9-withDind-v6`
+- `ubuntu22.04-node20.9-withDind-v7`
+- `ubuntu22.04-node20.9-withDind-v8`
```

Comment:
what happened with dind and node versions? are these out of sync?

Reply:
I would not talk about dind yet - it currently only works for Enterprises, and we can point them to it if they need it.

And it might confuse Pro users trying to use these images and wondering why Docker doesn't work.

I have an item this cycle to look at enabling this for all Pro users, so we can revisit then?

Reply:
removed dind from image list